src/main/java/org/apache/cassandra/sidecar/routes/StreamSSTableComponentHandler.java

```diff
@@ -64,43 +65,32 @@ public void handleInternal(RoutingContext context,
                                SocketAddress remoteAddress,
                                StreamSSTableComponentRequest request)
     {
-        validate(request)
```

Comment:
Removed validation since the validation is already happening at the `snapshotPathBuilder.build` step